docs/gke-installation.md

```diff
@@ -0,0 +1,126 @@
+# Deploying Antrea on a GKE cluster
+
+We support running Antrea inside of GKE clusters on Ubuntu host. Antrea would operate
+in noEncap mode, in which no encapsulation is required for any kind of traffic (Intra Node,
+Inter Node etc) and enforcing network policies using OVS.
```

Comment:
Need to revise the last sentence, like: and NetworkPolicies are enforced using OVS

Reply:
Done